vision.md

```diff
@@ -0,0 +1,115 @@
+
+## Who are our target users/personas?
+
+Within the different target segments, we are addressing the following personas with thin-edge.io:
+
+**(IoT) Solution Developer/ Solution Architect** (with background such as Python, Java, JS, Angular, Kubernetes, Cloud
+- Responsible for :
+	- Implementing and maintaining the end-to-end IoT solution
+	- Often juggling multiple initiatives covering a broad range of technology stacks in addition to implement and maintaining solutions.
+- Challenges and needs reagarding device enablement:
+	- Lack of expertise and knowledge in embedded space
+	- Dealing with fragmented hardware / linux variants
+	- Lack of time to focus on device enablement as building IoT applications on cloud side is main responsibility
+	- No interest/time to dive into “hygiene” factors as device management and security
+	- Expect ready to use or configuration based solution, with pre-defined design principles and framework, offering easy extensibility  with known tools/languages 
+
+**Device developer / Embedded engineer** with background in Linux, C/C++, C#, embedded systems (IT focused)
+
+- Responsible for
+	- device logic including firmware and software
+	- primary only tasked to connect one or many devices/types to overall IoT solution
+- Challenges and needs reagarding device enablement:
+	- enable new services and connectivity on the device while keeping stability and robustness (while having limited computing resources)
+	- dealing with certificates, queuing and persisting messages to handle instable connections, 
+	- allowing the device to be managed centrally, to keep it secure and up-to-date (while important to him not always #1 prio to overall initiative/project)
+
+- Special case : embedded dev with OT focus
+	- familiar with PLCs, SCADA systems,
+	- dealing with emerging need for connectivtiy and IIoT 
+	- key concern is security, robusteness and resource efficiency which is usually overruling any “typical” IT solution on the device and implies some kind of custom logic. (e.g. rather closed OS, no dependencies can be installed, very strict certification and QA process , no CI/CD possible, long prodcut lifecycle 10-20 years)
+
+**Summary:**
+
+The persona types adressed by thin-edge.io often have conflicting requirements and views, this itself is addressed by the project technology vision and design principles, allowing thin-edge.io acting as a brindge between the OT and IT world. 
```

Comment:
Please add a summary of what those conflicting requirements are. If they are fundamentally conflicting, then how can thin-edge still be useful to both? What solutions or tradeoffs would these personas have to make?